pkg/operation/botanist/vpnshoot.go

```diff
@@ -77,11 +68,6 @@ func (b *Botanist) DefaultVPNShoot() (vpnshoot.Interface, error) {
 func (b *Botanist) DeployVPNShoot(ctx context.Context) error {
 	secrets := vpnshoot.Secrets{}
 
-	if !b.Shoot.ReversedVPNEnabled {
-		dhSecret := b.getDiffieHellmanSecret()
-		secrets.DH = &dhSecret
-	}
-
```

Comment:
Now that we don't need this anymore, the whole `DeployVPNShoot` function seems useless. Drop it.